packages/shared/src/components/fields/Dropdown.tsx

```diff
@@ -162,7 +164,7 @@ export function Dropdown({
           )}
         />
       </button>
-      {isMobile ? (
+      {isMobile || openFullScreen ? (
```

Comment:
Can we maybe streamline that component get's the `openFullScreen` prop and then we can internally fallback it to true on mobile?

```js
const fullScreen = openFullScreen ?? isMobile
```

Just so we have single evaluation on component level and then also component will respect if anyone passes `openFullScreen={false}` no matter of resolution.

Reply:
Yup makes sense, changed 🙇‍♀️